packages/@adobe/spectrum-css-temp/components/modal/index.css

```diff
@@ -72,6 +72,7 @@ governing permissions and limitations under the License.
   /* Don't be bigger than the screen */
   max-height: var(--spectrum-dialog-max-height);
   max-width: 90vw;
+  min-width: var(--spectrum-dialog-min-width);
```

Comment:
Makes it so the modal cannot be smaller than the dialog. This stops the contents of the dialog from spilling out of the modal like so:
![image](https://user-images.githubusercontent.com/9661127/84213281-1943f100-aa75-11ea-9e6d-6b56a4f8bc48.png)

@dannify @devongovett This is what we (Rob, Kyle, and I) came up with after trying to change stuff in the dialog grid css to fix this, do you think its crazy?